flocker/acceptance/testtools.py

```diff
@@ -858,3 +859,34 @@ def clean(cluster):
             return calling_test_method
         return wrapper
     return decorator
+
+
+def create_dataset(test_case, cluster,
+                   maximum_size=REALISTIC_BLOCKDEVICE_SIZE):
```

Comment:
Should we have node number as a parameter? If so, this method could be reused to seed move tests in future? Thanks!